cfe_internal/enterprise/federation/federation.cf

```diff
@@ -165,12 +170,16 @@ bundle agent transport_user
       edit_template_string => "{{#-top-}}{{{.}}}$(const.n){{/-top-}}",
       template_data => @(cfengine_enterprise_federation:config.fingerprints),
       template_method => "inline_mustache";
+    "$(ssh_config)"
+      create => "true",
+      handle => "ssh_config_configured",
```

Comment:
I would probably call this ssh_configuration_identityfile since that is the only thing it's managing. 

Maybe edit_defaults => empty as well? If there is something in the file then IdentityFile might not get applied by default right?

Reply:
> I would probably call this ssh_configuration_identityfile since that is the only thing it's managing.

Well, it's still a config file. We may add more things there in the future.

Reply:
> Maybe edit_defaults => empty as well? If there is something in the file then IdentityFile might not get applied by default right?

Yes, but then we want things to work that way (e.g. a different key for a specific host).